doc/pandas.rst

```diff
@@ -125,3 +125,67 @@ xarray objects do not yet support hierarchical indexes, so if your data has
 a hierarchical index, you will either need to unstack it first or use the
 :py:meth:`~xarray.DataArray.from_series` or
 :py:meth:`~xarray.Dataset.from_dataframe` constructors described above.
+
+
+Transitioning from pandas' Panels to xarray
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+[Preamble about deprecating Panel to xarray etc]
```

Comment:
Panel, pandas's data structure for 3D arrays, has always been a second class data structure compared to the `Series` and `DataFrame`. To allow pandas developers to focus more on its core functionality built around the DataFrame, they recently deprecated Panel (as of pandas v0.19, released on YYYY-MM-DD) in favor of xarray.